src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -29,6 +29,7 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
 
         String preppedWord = word.trim();
         checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
+        System.out.println(preppedWord);
```

Comment:
Remember to remove the System.out.println!

Reply:
Whoops, done!